InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java

```diff
@@ -6,36 +6,61 @@
 import frc.robot.subsystem.drive.VelocityDrive;
 import frc.robot.subsystem.drive.DriveSubsystem.DriveMethod;
 import frc.robot.utils.CommandUtils;
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj.trajectory.Trajectory.State;
 
 
 
 public class AutoDrive extends RamseteCommand {
     private final DriveSubsystem DRIVE_SUBSYSTEM;
 
+    private Timer timer;
```

Comment:
Isn't it obnoxious that WPI made their RamseteCommand timer private so you can't access it?

Reply:
ikr? and I can't even extend it because it's not private :/

Reply:
protected*